docs/parsers/Win2d_Path_Mini_Language.md

```diff
@@ -0,0 +1,872 @@
+---
+title:
+author: ratishphilip
+description: describe your control in one sentence
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, win2d, win2d path mini language
+dev_langs:
+  - csharp
+---
+
+- [Introduction](#introduction)
+  - [Requirements](#requirements)
+- [Win2d Path Mini Language](#win2d-path-mini-language)
+  - [Path Mini Language](#path-mini-language)
+    - [Fill Behavior](#fill-behavior)
+    - [MoveTo](#moveto)
+    - [LineTo](#lineto)
+    - [Horizontal LineTo](#horizontal-lineto)
+    - [Vertical LineTo](#vertical-lineto)
+    - [Cubic Bézier](#cubic-bézier)
+    - [Smooth Cubic Bézier](#smooth-cubic-bézier)
+    - [Quadratic Bézier](#quadratic-bézier)
+    - [Smooth Quadratic Bézier](#smooth-quadratic-bézier)
+    - [Arc](#arc)
+    - [Close Path](#close-path)
+    - [Ellipse Figure](#ellipse-figure)
+    - [Polygon Figure](#polygon-figure)
+    - [Rectangle Figure](#rectangle-figure)
+    - [RoundedRectangle Figure](#roundedrectangle-figure)
+  - [CanvasBrush Mini Language](#canvasbrush-mini-language)
+    - [ICanvasBrush Attribute Commands](#icanvasbrush-attribute-commands)
+      - [Start Point](#start-point)
+      - [End Point](#end-point)
+      - [Opacity](#opacity)
+      - [Alpha Mode](#alpha-mode)
+      - [Buffer Precision](#buffer-precision)
+      - [Edge Behavior](#edge-behavior)
+      - [Pre Interpolation Color Space](#pre-interpolation-color-space)
+      - [Post Interpolation Color Space](#post-interpolation-color-space)
+      - [Origin Offset](#origin-offset)
+      - [GradientStop](#gradientstop)
+      - [GradientStopHdr](#gradientstophdr)
+    - [SolidColorBrush](#solidcolorbrush)
+    - [LinearGradientBrush](#lineargradientbrush)
+    - [LinearGradientBrush with GradientStopHdr](#lineargradientbrush-with-gradientstophdr)
+    - [RadialGradientBrush](#radialgradientbrush)
+    - [RadialGradientBrush with GradientStopHdr](#radialgradientbrush-with-gradientstophdr)
+  - [CanvasStrokeStyle Mini Language](#canvasstrokestyle-mini-language)
+    - [CanvasStrokeStyle Attributes](#canvasstrokestyle-attributes)
+      - [Dash Style](#dash-style)
+      - [Line Join](#line-join)
+      - [Miter Limit](#miter-limit)
+      - [Dash Offset](#dash-offset)
+      - [Start Cap](#start-cap)
+      - [End Cap](#end-cap)
+      - [Dash Cap](#dash-cap)
+      - [Transform Behavior](#transform-behavior)
+      - [Custom Dash Style](#custom-dash-style)
+    - [Defining the CanvasStrokeStyle](#defining-the-canvasstrokestyle)
+  - [CanvasStroke Mini Language](#canvasstroke-mini-language)
+    - [ICanvasStroke interface and CanvasStroke class](#icanvasstroke-interface-and-canvasstroke-class)
+- [Creating Geometries, Brushes, Strokes and StrokeStyles](#creating-geometries-brushes-strokes-and-strokestyles)
+
+# Introduction
```

Comment:
You can't have multiple top-level headings, be sure to check the warnings from the build comment.